ops/reth-cross-test/phases/fault_phase.py

```
@@ -0,0 +1,202 @@
+    def _wipe_reth_sequencer(self) -> dict:
+        """Wipe morph-geth-0 data volume, restart, verify it re-syncs."""
+        print("Fault: Wiping reth sequencer (morph-geth-0) data and re-syncing...")
+        w3_other = Web3(Web3.HTTPProvider(config.SEQUENCER_RPCS["morph-geth-1"], request_kwargs={"timeout": 10}))
+        try:
+            target_height = w3_other.eth.block_number
+        except Exception:
+            target_height = 0
+
+        # Stop, wipe volume, restart
+        self._docker_cmd("stop", "morph-geth-0")
+        subprocess.run(["docker", "volume", "rm", "-f", "morph_data_0"], check=False)
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)